scheduler/licenses/license.txt

```diff
@@ -531,27 +531,27 @@ Azure/go-autorest  Apache License 2.0  https://github.com/Azure/go-autorest/blob
 --------------------------------------------------------------------------------
 AzureAD/microsoft-authentication-library-for-go  MIT License  https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/main/LICENSE
 --------------------------------------------------------------------------------
-    MIT License

```

Comment:
this is an autogenerated file, please revert.

Reply:
just reversed the changes. Please recheck